### DISCUSSION TITLE
Seeking more install detail for newbie

### DISCUSSION THREAD

**Discussion by gallaugher** [YAtechnologies/GoogleMaps-SP#8] [General]
Hi! Thanks so much for adding this. 
I want to use GooglePlaces Autocomplete with my students & in each class there are ALWAYS students that have issues with Cocoapods, so I'm hoping package manager works. Apologies, but for me it's unclear how to use this. The package doesn't seem to install with the simple File > New Package > paste in the git URL. It also says it needs Carthage. Does anyone have more detailed, newbie-oriented instructions for this?
Thanks!

**Comment by karimhm** [comment]
Carthage is only needed for compiling the package [XCFramework](https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages) files. The following guide might help: [_Adding Package Dependencies to Your App_](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app)

**Reply by gallaugher** [reply]
Thanks so much. Figured it out. In case anyone else is a newbie at this I put together a tutorial on installation. https://youtu.be/GiFKZixb6vE

**Reply by karimhm** [reply]
@gallaugher Thanks for the video. One thing I want to note is that **Carthage** is only needed if you want to create your private version of the package (hosting it on your account). For installation, you only need Xcode 12 or later nothing else.
